Added cantons of Angola

Zebedeu opened the pull request:
Added cantons of Angola.  All provinces or states of Angola have been added.

Zebedeu committed:
Added cantons of Angola

claudiosanches commented:
Can you provide some source for the ISO codes?
I could just find this one: https://en.wikipedia.org/wiki/ISO_3166-2:AO
But is using 3 letters and not 2... Need a source just for future reference and make sure all is correct.
Thanks for your contribution.

Zebedeu committed:
fix cantons of Angola

fix cantons of Angola. Angola (ISO Country code)

Subdivision names are listed as in the ISO 3166-2 standard published by
the ISO 3166 Maintenance Agency (ISO 3166/MA).

source: https://en.wikipedia.org/wiki/ISO_3166-2:AO

Zebedeu commented:
I found the following sources referring to the ISO of Angola and its respective provinces or states.

http://www.statoids.com/uao.html

http://194.132.104.143/GROUP/Standard/ACCodes.nsf/baf0033777518616c1256f43007bc4f2/0f6321291836c6bcc1256a11002665b9?OpenDocument

//en.wikipedia.org/wiki/ISO_3166-2:AO

mikejolley commented:
@Zebedeu Are there any eCommerce stores base din Angola and do they use a dropdown of states or allow text input? What is more common?

Zebedeu commented:
The shops and commerce in angola is a trend. And the stores use a dropdown menu for the states or provinces we designate #16059 
![captura de ecra 2017-07-12 as 12 22 06](https://user-images.githubusercontent.com/12781593/28116086-65ab6f8a-66ff-11e7-916a-c26f49a17fb1.png)
![captura de ecra 2017-07-12 as 12 30 12](https://user-images.githubusercontent.com/12781593/28116087-65b01c74-66ff-11e7-84ee-30dc96f8a790.png)

mikejolley commented on i18n/states/AO.php:
```diff
@@ -0,0 +1,35 @@
+$states['AO'] = array(
+		'BGO' => __( 'Bengo', 'woocommerce' ),
+	    'BLU' => __( 'Benguela', 'woocommerce' ),
+	    'BIE'  => __( 'Bié', 'woocommerce' ),
```
The items here and below have extra spacing before the `=>`

Zebedeu replied:
Yes ... it was an indentation error. Will be corrected now.

Zebedeu committed:
fix indentation

Zebedeu committed:
fix indentation

mikejolley commented:
@Zebedeu There are still 2 spaces before the `=>` on all the lines.

Zebedeu committed:
fix indentation update

mikejolley commented:
It was still very off in terms of indenting. Merged https://github.com/woocommerce/woocommerce/pull/16092
